frontend/src/modules/_shared/InplaceVolumetrics/queryHooks.ts

```diff
@@ -0,0 +1,225 @@
+import {
+    FluidZone_api,
+    InplaceStatisticalVolumetricTableDataPerFluidSelection_api,
+    InplaceVolumetricResultName_api,
+    InplaceVolumetricTableDataPerFluidSelection_api,
+    InplaceVolumetricsIdentifierWithValues_api,
+    InplaceVolumetricsIdentifier_api,
+} from "@api";
+import { apiService } from "@framework/ApiService";
+import { EnsembleIdent } from "@framework/EnsembleIdent";
+import {
+    InplaceVolumetricsStatisticalTableData,
+    InplaceVolumetricsTableData,
+} from "@modules/_shared/InplaceVolumetrics/types";
+import { UseQueryResult } from "@tanstack/react-query";
+
+export type EnsembleIdentWithRealizations = {
+    ensembleIdent: EnsembleIdent;
+    realizations: readonly number[];
+};
+
+const STALE_TIME = 60 * 1000;
+const CACHE_TIME = 60 * 1000;
```

Comment:
I totally agree that it gets a bit out of hand. However, we haven't decided yet which cache/stale times to use for which type of data. I think this could be a good topic for the workshop.

Reply:
I'll keep it as it is for now